cloudigrade/util/aws.py

```diff
@@ -8,6 +8,7 @@
 import boto3
 from botocore.exceptions import ClientError
 from django.conf import settings
+from django.core.exceptions import SuspiciousOperation
```

Comment:
Not sure if `SuspiciousOperation` really feels right here, even if we're just re-using it for the easy `400` we may inadvertently accidentally catch `InvalidArn` exceptions if we're catching `SuspiciousOperation` in the future when trying to catch actual security issues. I'd encourage you to extend [DRF `ApiException`](http://www.django-rest-framework.org/api-guide/exceptions/#api-reference) instead, maybe [ParseError](http://www.django-rest-framework.org/api-guide/exceptions/#parseerror).